scripts/jenkins/cloud/ansible/install-crowbar.yml

```diff
@@ -53,6 +53,28 @@
           loop_control:
             loop_var: command
 
+        - block:
+            - name: Get sync_mark json from data bag
+              command: "knife data bag show crowbar-config sync_mark -F json"
+              register: _sync_mark_json
+
+            - name: Update timeout_multiplier value
+              set_fact:
+                _sync_mark_updated: "{{ _sync_mark_json.stdout | regex_replace(
```

Comment:
why not `_sync_mark_updated: "{{ _sync_mark_json.stdout | from_json | combine({ 'timeout_multiplier': versioned_features.crowbar_sync_mark_timeout_update.value }) }}"` ?

Reply:
done. thanks!